client/src/components/App/App.js

```diff
@@ -15,8 +26,20 @@ const App = () => {
         weights={weights}
         setWeights={setWeights}
       />
-      <Weights weights={weights} setWeights={setWeights} />
-      {/* <Results /> */}
+      <Button variant="primary" onClick={getChartData}>
+        Render Chart
+      </Button>
+      <br />
+      <br />
```

Comment:
instead of 2 br maybe padding is better? Up to you but i think 2 br looks ugly

Reply:
I think your face is ugly